docker-compose.yml

```diff
@@ -40,6 +41,8 @@ services:
     shm_size: 2g # 2 Gig seems like a good size
     volumes:
       - .:/app
+      - ${GCLOUD_DIR:-.}:/home/.config/gcloud
```

Comment:
This looks unused.

Reply:
This is for local telemetry alert testing. We can't use the `GOOGLE_APPLICATION_CREDENTIALS` setting there since we don't have service accounts. We have to login with `gcloud auth ...`, and then pass the authentication folder through to the docker using this variable.